src/styles/inc/_vars.scss

```diff
@@ -0,0 +1,29 @@
+@use "sass:math" as *;
+@use "sass:list" as *;
+
+$white: #fff;
+$main: #060b35;
+$second: #616070;
+$blue: #00acdc;
+$bodyfont: "Roboto", sans-serif;
+$padding-middle: 24px;
+
+// <Mixins-media-queries>=========================================================================================================
+
+@function vw($value) {
+  $vwvalue: calc(#{$value} / #{$maxwidth} * 100vw);
+
+  @return $vwvalue;
+}
+
+@function rem($value) {
+  $remvalue: div($value, 16) + rem;
```

Comment:
now you always will have 16 its not make sense if you not using variable instead

Reply:
Understand, but the idea in that mixin is 16 like the default font size in the browser